tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "allowSyntheticDefaultImports": true,
-    "noUnusedParameters": true,
+    "noUnusedParameters": false,
```

Comment:
> noUnusedParameters
This parameter is for managing the unused variable in the functions in typescript file. by setting it false means it will not give any warning and app compilation issue. While it is true it gives compilation error of the angular application until not resolved.